src/components/custom-field-input-date/custom-field-input-date.md

```diff
@@ -0,0 +1,61 @@
+A simple date field for custom fields. Permits a variety of formats and allows custom validation.
+
+##### Basic usage:
+
+```jsx
+<CustomFieldInputDate
+    id="my-birthday-1"
+    label="Andre's Birthday" 
+    value="1992-05-10"
+/>
+
+<CustomFieldInputDate
+    id="my-birthday-2"
+    label="Greyson's Birthday"
+    value="08/20/2020"
+/>
+```
+----
+
+##### Disabled state:
+
+```jsx
+<CustomFieldInputDate
+    id="my-birthday-4"
+    label="Juanca's Birthday"
+    value="03/03/1990" 
+    disabled
+/>
+```
+----
+
+##### Errored state:
+
+Below are two examples of an invalid custom field input: a syntactic error and a semantic error. The first below is
```

Comment:
We should probably just let the browser wrap these lines instead of newlining them, unless we want them separated by whitespace for clarity.